src/font.rs

```diff
@@ -264,7 +264,13 @@ impl Font {
         }
 
         if let Some(features) = self.features.as_ref() {
-            fs::write(path.join(FEATURES_FILE), features)?;
+            // Normalize feature files with line feed line endings
+            // This is consistent with the line endings serialized in glif and plist files
+            if features.contains('\r') {
+                fs::write(path.join(FEATURES_FILE), features.replace("\r", ""))?;
```

Comment:
I think VTT table dumps (in fontTools' TTX format) contain `\r`, but the two characters and not `\u000D`. I think the data folder shouldn't be touched anyway.

Reply:
Oh wait. It actually contains literal `\u000D`, see https://github.com/daltonmaag/ubuntu/blob/master/source/Ubuntu-B.ufo/data/com.github.fonttools.ttx/T_S_I__1.ttx. Maybe because VTT started life on very old Macs that still used `\r`?

I suppose mixing LF and CRLF in any part of the UFO (glyph lib or elsewhere) is a bad idea so I guess this PR is fine, though.

Reply:
The data will be read in as a byte array?